src/scripts/constants.ts

```diff
@@ -161,10 +165,12 @@ export module Constants {
 		export var pdfHeaderTitle = "pdfHeaderTitle";
 		export var pageRangeControl = "pageRangeControl";
 
+		// pdfClipOptions
 		export var checkboxToDistributePages = "checkboxToDistributePages";
 		export var checkboxToAttachPdfDisabled = "checkboxToAttachPdfDisabled";
 		export var checkboxToAttachPdf = "checkboxToAttachPdf";
 		export var moreClipOptions = "moreClipOptions";
+		export var popover = "popover";
```

Comment:
#popover doesn't look like it's being used to style anything. Do we still need it as an id? I'd rather this be just a class so we can have better re-use.